rfcs/shared/build-system/triage-automation.md

```diff
@@ -0,0 +1,161 @@
+
+This section is important as the motivation or problem statement is indepenent from the proposed change. Even if this RFC is not accepted this Motivation can be used for alternative solutions.
+
+In the end, please make sure to present a neutral Problem statement, rather than one that motivates a particular solution
+-->
+
+## Detailed Design or Proposal
+
+<!-- This is the bulk of the RFC. Explain the proposal or design in enough detail for the inteded audience to understand. -->
+
+After we [introduced new github issues beta](https://github.com/microsoft/fluentui/pull/22337) for reporting issues and feature request, the experience for both contributors and shield devs was significantly improved. A followup that was discussed is to implement further automation by leveraging Github Actions, This document will describe the approach in better detail.
+
+We would like to automate following steps:
+
+1. project label assignment
+2. package label assignment
+3. assign issue to project board/team
+
+**Pre-requirements:**
+
+- normalization/unification of labels within monorepo for projects
+
+_Current state:_
+
+- 2 labels for v0
+  - https://github.com/microsoft/fluentui/labels?q=northstar
+- 2 labels for v9
+  - https://github.com/microsoft/fluentui/labels/Fluent%20UI%20react-components%20%28v9%29
+  - https://github.com/microsoft/fluentui/labels/Fluent%20UI%20vNext
+- per component labels
+  - same labels mixed for various fluent version https://github.com/microsoft/fluentui/labels/Component%3A%20Dialog
+
+_Expected state:_
+
+- 1 label for v0
+- 1 label for v9
+- label per published package
+- "component labels" can stay as are but should be used solely for v8 (internal react package "sub-packages" https://github.com/microsoft/fluentui/labels/Component%3A%20Dialog)
+
+### 1. project label assignment
+
+> Note:
+>
+> - by "project" we mean one of libraries that exist within monorepo (v0,v8,v9,web-components)
+
+**Workflow:**
+
+Based on selected project within issue template, we should add appropriate label.
+
+<img width="1108" alt="image" src="https://user-images.githubusercontent.com/1223799/190106473-fd007d87-f09a-4493-aafd-781a8404ae45.png">
+
+> NOTE:
+>
+> - this is already implement via [github action](https://github.com/microsoft/fluentui/pull/24788)
+
+### 2. package label assignment
+
+> Note:
+>
+> - by "package" we mean official package name used to publish to npm registry without `@scope` prefix
+
+**Workflow:**
+
+Based on selected project within issue template, additional select should be dynamically generated so contributor can pick proper package.
+
+<img width="569" alt="example of flow" src="https://user-images.githubusercontent.com/1223799/190431945-56fcd343-6eb7-4145-b2b2-62f5919b8c31.png">
+
+Unfortunately **[dynamic input controls are not possible with github issues beta](https://github.com/community/community/discussions/29067)** which leaves us with following options how to implement this:
+
```

Comment:
Does this allow you to select multiple packages? Or in case the issue affected multiple packages, how would the scenario proceed?

Reply:
good question, we can enable multiple items selected with github forms https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema#dropdown. 

What would be the use-case ?

**to provide more context:**

Our template is aimed mainly for 3rd party with workflow.

Meaning: "Im using fluent, Im using this component and it doesn't work for me."

From my experience user reports issue for the immediate thing that comes in his way( component A ). 

Additionally one can dig deeper and find that the issue is within some underlying component (Component B) - thus the reporter should pick that one.

In general having fine grained issue reports reduces the complexity and time to identify the problem. This is omnipresent everywhere - dissecting task to smaller ones/ atomic commits/small PR's.

to wrap up I'm IMO it shouldn't be needed. but as mentioned the option is there